frontend/src/modules/projects/views/projectView.js

```diff
@@ -76,14 +76,23 @@ define(function(require) {
     deleteProjectPrompt: function(event) {
       event && event.preventDefault();
       if(this.model.get('_isShared') === true) {
-        Origin.Notify.confirm({
-          type: 'warning',
-          title: Origin.l10n.t('app.deletesharedproject'),
-          text: Origin.l10n.t('app.confirmdeleteproject') + '<br/><br/>' + Origin.l10n.t('app.confirmdeletesharedprojectwarning'),
-          destructive: true,
-          callback: _.bind(this.deleteProjectConfirm, this)
-        });
+        if(this.model.get('createdBy') === Origin.sessionModel.id){
+          Origin.Notify.confirm({
+            type: 'warning',
+            title: Origin.l10n.t('app.deletesharedproject'),
+            text: Origin.l10n.t('app.confirmdeleteproject') + '<br/><br/>' + Origin.l10n.t('app.confirmdeletesharedprojectwarning'),
+            destructive: true,
+            callback: _.bind(this.deleteProjectConfirm, this)
+          });
+        } else {
+          Origin.Notify.alert({
+            type: 'error',
+            title: Origin.l10n.t('app.error'),
```

Comment:
There shouldn't be any need for this, as a default title is added here:
- [modules/notify/plugins/alert/index.js#L32](https://github.com/adaptlearning/adapt_authoring/blob/master/frontend/src/modules/notify/plugins/alert/index.js#L32)
- [en-application.json#L248](https://github.com/adaptlearning/adapt_authoring/blob/master/routes/lang/en-application.json#L248)